tests/e2e/tests/020_packages/010_unauth_publish.ts

```diff
@@ -0,0 +1,33 @@
+import { npmPublish } from '../../utils/process';
+import * as fs from '../../utils/fs';
+
+export default function() {
+  return Promise.resolve()
+    .then(() => {
+      return new Promise((resolve, reject) => {
```

Comment:
no need to return a `new Promise`. just chain it further from previous one. this doesn't look nice.